### ISSUE TITLE
I am experiencing an issue with Wezterm where it creates files like `gui-sock-12888` in the `~/.local/share/wezterm` directory even though my config file `wezterm.lua` is located in `~/.config/wezterm`.

### ISSUE THREAD

**Issue by khanra17** [wez/wezterm#4421]
**Describe the issue**

I have set up my Wezterm configuration in accordance with the XDG Base Directory Specification by placing my `wezterm.lua` file in the `~/.config/wezterm` directory and it works great. However, when I open the terminal, Wezterm creates files like `gui-sock-12888` in the `~/.local/share/wezterm` directory.

I have also attempted to set the `XDG_RUNTIME_DIR` environment variable, but this did not resolve the issue.

If there is a way to configure Wezterm to use a different directory for these files via environment variables or any other workaround, please let me know.

Thank you for your assistance in resolving this issue.

**Comment by arodland** [comment]
That would be `XDG_DATA_HOME`. Config is for config files.

**Comment by khanra17** [comment]
> That would be `XDG_DATA_HOME`. Config is for config files.

No. even after setting `XDG_DATA_HOME` its still making `gui-sock-**` files in `~\.local\share\wezterm\`

**Comment by random-donuts** [comment]
Same here, it seems to not follow the XDG_DATA_HOME variable
